help/wscript

```diff
@@ -74,11 +80,12 @@ TaskGen.after('init_mallard')(apply_mallard)
 
 
 def configure(ctx):
-	lookfor = "xml2po"
+	ctx.find_program('msgfmt', var='MSGFMT')
```

Comment:
The below check for `itstool` crashes and gently disables documentation, but the check for `msgfmt` will stop the build. Is that intended? I'm not even sure why it should be normal to build without the translations, since it's clearly stated as a build dependency in the README -- not sure if other assumptions break later on, anyways?

Seems that help, documentation and translation is a bit mixed up, too.

Could the whole exception handling below be removed or does it have a reason to be?

Reply:
I don't have too much experience with waf. Now I moved it inside the exception handling.

Reply:
Thanks for addressing it :+1: